src/app/app.component.css

```diff
@@ -0,0 +1,10 @@
+mat-card-content {
+  display: flex;
+  flex-direction: row;
+  padding: 0.5rem;
+}
+
+.component {
```

Comment:
Does this need to be updated so the UI looks the same with the class name changes you made?

Reply:
Yes just fixed in last commit.